### PR TITLE
remove game script

### PR DESCRIPTION
**Pull request by pwgraham91**

### CODE REVIEW

File: app/scripts/remove_game.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+def replay_game(session, next_game, affected_player_ids):
+    winner = next_game.winner
+    loser = next_game.loser
+    if next_game.winner_id in affected_player_ids:
+        winner_pre_elo = winner.elo
+    else:
+        winner_previous_game = get_previous_game(session, winner, next_game)
+        if winner_previous_game:
+            winner_pre_elo = winner_previous_game.winner_elo_score if winner_previous_game.winner_id == winner.id else winner_previous_game.loser_elo_score
```
Comment (pwgraham91) [inline app/scripts/remove_game.py]: use get_score_from game func

File: app/scripts/remove_game.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        else:
+            winner_pre_elo = 100
+    if next_game.loser_id in affected_player_ids:
+        loser_pre_elo = loser.elo
+    else:
+        loser_previous_game = get_previous_game(session, loser, next_game)
+        if loser_previous_game:
+            loser_pre_elo = loser_previous_game.winner_elo_score if loser_previous_game.winner_id == loser.id else loser_previous_game.loser_elo_score
```
Comment (pwgraham91) [inline app/scripts/remove_game.py]: use get_score_from_game func

**Review by youngrichard — APPROVED** [review]
Looks good to me code-wise. Should test a few times. Feel free to delete my losses for the cause.

**Comment by pwgraham91** [comment]
@youngrichard as far as testing goes, I made a pull prod script so I was able to test it locally. I'm confident it works

**Pull request merged**